lib/mix/lib/mix/tasks/release.ex

```diff
@@ -123,6 +123,63 @@ defmodule Mix.Tasks.Release do
   You can also use `remote` to connect a remote IEx session to the
   system.
 
+  As you operate your system, you may find yourself running some piece of code
+  as a one-off command quite often and re-typing the whole command every
+  time may become cumbersome and error prone. For example, suppose you need to
+  regularly migrate your database and occasionally build a report of currently
+  connected users. You may consider creating a module to group these tasks:
+
+      # lib/my_app/release_tasks.ex
+      defmodule MyApp.ReleaseTasks do
+        def migrate_db() do
+          # ...
+        end
+
+        def print_connected_users() do
+          for user <- connected_users() do
+            # IO.puts ...
+          end
+        end
+      end
+
+  Again, for migrating the database we'd probably use `eval` but to list
+  currently connected users we'd probably connect to a running system using
+  `rpc`:
+
+      bin/RELEASE_NAME eval "MyApp.ReleaseTasks.migrate_db()"
+      bin/RELEASE_NAME rpc "MyApp.ReleaseTasks.print_connected_users()"
+
+  Finally, you may still automate this further by creating standalone scripts
```

Comment:
The section below has probably way too much code so maybe it's better to show the functions but let users fill in the blanks, or skip this thing altogether?

Reply:
Yeah, we should probably keep this focused. I have pushed some updates, let me know what you think.